src/QmlControls/FactSliderPanel.qml

```diff
@@ -110,6 +111,26 @@ Column {
                         onValueChanged: {
                             if (_loadComplete) {
                                 fact.value = value
+                                    }
+                                }
+
+                                updateValueWhileDragging:   true
```

Comment:
Not sure this is a good idea. On small screens like a phone, dragging these things around can be a bit hard. Hence you hit a number of bad values while you are trying to get to the final right place. You don't want the intermediate values to make it to the vehicle.